apps/tsconfig.json

```diff
@@ -24,7 +24,9 @@
     "allowJs": true,
     "baseUrl": "./",
     "paths": {
-      "@cdo/apps/*": ["./src/*"]
-    }
+      "@cdo/apps/*": ["./src/*"],
+      "@cdo/static/*": ["./static/*"],
```

Comment:
Ah, is this so that we can do an `import` from `@cdo/static`?

Reply:
Yes, although I saw after adding this you already had a way to do this. Seems like a positive to have import work with these static paths in TS though anyway)?

`const inputLibrary = require(`@cdo/static/dance/ai/${inputLibraryFilename}.json`);`

Reply:
I suspect so.  @molly-moen or @sanchitmalhotra126 probably know for sure.

Reply:
Is there any follow-up action item to this?

Reply:
I don't think so, think it was more of a clarification

Reply:
I'd say `import` is preferred if it works - using `require` allows us to circumvent TypeScript path aliasing which is helpful for certain files with specific knowns aliases (like @cdo/locale) but it removes any type checking and is basically an escape hatch to defer module resolution to webpack at runtime.